.github/workflows/release.yml

```diff
@@ -122,3 +149,41 @@ jobs:
           token: ${{ secrets.ELEVATED_GITHUB_TOKEN}}
           inputs: '{"build-ref": "${{ steps.commit-change-push.outputs.build-ref }}", "make-prerelease": "false"}'
           ref: ${{ needs.prepare-release.outputs.build-ref }}
+
+      - name: Revert notification channel
+        if: ${{ github.event.inputs.notification-channel != '' }}
+        run: |
+          sed -i.bak -e 's|\(notification_channel * = *"\)[^"]*|\1${{ steps.notification-channel.outputs.prev-channel }}|g' .release/ci.hcl
+          rm -rf .release/ci.hcl.bak
+          git diff --color=always .release/ci.hcl
```

Comment:
We're calling this "revert" but it's not obvious whether this literally backing out "Generate files for..." commit or making a slightly different change that could potentially drift. Could we literally reset this file with git (ex. `git reset $ref -- .release/ci.hcl`)?

This is particularly the case for the step below too, where we're doing two different things.

Reply:
Ah yeah, good point. This one is direct revert, the next one not quite, so I will rename it as well. Thanks!